### PR TITLE
Feature 10296 deactivate channels

### PR DESCRIPTION
**Pull request by Alfro**
Code review.

### CODE REVIEW

File: src/Pumukit/NewAdminBundle/Controller/SeriesController.php [hunk 1]
```diff
@@ -135,6 +135,13 @@ public function updateAction(Request $request)
 
         $pubDecisionsTags = $factoryService->getTagsByCod('PUBDECISIONS', true);
 
+        //These fields are form fields that are rendered separately, so they should be 'excluded' from the generic foreach.
+        //FIXME: There probably is a cleaner approach.
+        $exclude_fields = ['pumukitnewadmin_series_i18n_header', 'pumukitnewadmin_series_i18n_footer', 'pumukitnewadmin_series_i18n_line2', 'pumukitnewadmin_series_template'];
```
Comment (rubenrua) [inline src/Pumukit/NewAdminBundle/Controller/SeriesController.php]: Don't use the new array notation.

Comment (Alfro) [inline src/Pumukit/NewAdminBundle/Controller/SeriesController.php]: It rocks, though.

Comment (rubenrua) [inline src/Pumukit/NewAdminBundle/Controller/SeriesController.php]: https://github.com/campusdomar/PuMuKIT2/blob/master/composer.json#L11

PHP 5.3 doesn't support it.

File: src/Pumukit/NewAdminBundle/Resources/views/Series/updatemeta.html.twig [hunk 2]
```diff
@@ -31,8 +31,7 @@
     <div>
       <fieldset>
         <div class="col-md-7">
-
-	      {% for child in form if (child.vars.id not in ['pumukitnewadmin_series_i18n_header', 'pumukitnewadmin_series_i18n_footer', 'pumukitnewadmin_series_i18n_line2', 'pumukitnewadmin_series_template']) %}
+	  {% for child in form if (child.vars.id not in exclude_fields) %}
```
Comment (mreyk) [inline src/Pumukit/NewAdminBundle/Resources/views/Series/updatemeta.html.twig]: Not enough to not show the Series Type field (Channel). It is shown at the end of the form (line 87).

**Pull request merged**